js/addRowToGrid.js

```diff
@@ -811,10 +811,12 @@ var addMediaRowOnClick;
       if (finerSrefIsPrecise) {
         precisionCtrl.removeAttr('disabled');
         precisionCtrl.removeAttr('placeholder');
+        precisionCtrl.addClass('grid-required {precisionConditionalMandatory:true}');
```

Comment:
Can you set the class to `{pattern:/^[1-9][0-9]*$/, required:true}` and therefore get around the need to create a custom validation function? Also I don't think grid-required actually does anything - the only place I can find this class has an effect is on the Seasearch form.

Reply:
Thanks @johnvanbreda - I didn't really appreciate how that worked. That's much neater. I have made the changes and pushed the branch again.